pkg/kubectl/cmd/describe.go

```diff
@@ -81,7 +82,7 @@ func NewCmdDescribe(f cmdutil.Factory, out, cmdErr io.Writer) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "describe (-f FILENAME | TYPE [NAME_PREFIX | -l label] | TYPE/NAME)",
-		Short:   i18n.T("Show details of a specific resource or group of resources"),
+		Short:   i18n.T("Show details of a specific resource or group of resources, include the uninitialized objects by default, unless user explicitly set --include-uninitialized=false"),
```

Comment:
What I was saying is leave the short help text as it used to be. Only add related info in long help text is good enough.
In other words, please drop the change in this line.